tests/test_pytorch/test_deepspeed.py

```diff
@@ -99,7 +99,7 @@ def get_device_count(pytestconfig):
 @pytest.fixture()
 def deepspeed_base_config():
     return {
-        "train_batch_size": 8,
+        "train_batch_size": 2,
```

Comment:
```suggestion
        "train_batch_size": 12,
```
this can be any number, right so, rather do not correlate it with nb devices?